assets/app/views/catalog/images.html

```diff
@@ -11,7 +11,8 @@
           <h1>Select a builder image</h1>
         </div>
       </div>
-      <div ng-if="!builders.length">There are no builder images to select from.  To add a builder to your project run <code>osc create -f &lt;image_repository_file&gt; -n {{projectName}}</code></div>
+      <div ng-if="!loadedProjectImageStreams && !loadedSharedImageStreams">Loading...</div>
```

Comment:
@liggitt Maybe this is why you asked, but the behavior is a little different. Right now the "Loading..." message stays until builders from both namespaces load. This will remove the message as soon as any builders are loaded, but not necessarily all.